python/paddle/v2/framework/tests/test_word2vec.py

```diff
@@ -108,8 +129,8 @@
     init_program=init_program)
 avg_cost = layers.mean(x=cost, program=program, init_program=init_program)
 
-sgd_optimizer = optimizer.SGDOptimizer(learning_rate=0.001)
-opts = sgd_optimizer.minimize(avg_cost)
+adagrad_optimizer = optimizer.AdagradOptimizer(learning_rate=0.001)
```

Comment:
This will not work till https://github.com/PaddlePaddle/Paddle/issues/4975 is fixed. @jacquesqiao is working on this. So please do not merge this PR before the issue is fixed.